akka-docs/src/main/paradox/logging.md

```diff
@@ -389,15 +389,15 @@ One gotcha is that the timestamp is attributed in the event handler, not when ac
 
 The SLF4J logger selected for each log event is chosen based on the
 @scala[`Class[_]`] @java[`Class`] of the log source specified when creating the
-`LoggingAdapter`, unless that was given directly as a string in which
+@apidoc[LoggingAdapter], unless that was given directly as a string in which
```

Comment:
Which `LoggerFactory` should I use? `org.slf4j.LoggerFactory` or `com.typesafe.sslconfig.util.LoggerFactory`? I'm not sure :(

Reply:
That's `org.slf4j.LoggerFactory`